### PR TITLE
Motion detection 

### PR DESCRIPTION
**Pull request by kskrao219**
# Description

Added the script which detects the motion in front of the camera.

Fixes #(#1191)

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Python/Motion_Detection/script.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    cv2.putText(img, text, (10, 20), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 255), 2)
+
+    cv2.imshow("cameraFeed", img)
+    
+    key = cv2.waitKey(1) & 0xFF
+    if key == ord("q"):
+        break
+cam.release()
+cv2.destroyAllWindows()
+
```
Comment (HarshCasper) [inline Python/Motion_Detection/script.py:47]: Remove the extra line

File: Python/Motion_Detection/script.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        if cv2.contourArea(c) < area:
+            continue
+        # draw the rectangle
+        (x, y, w, h) = cv2.boundingRect(c)
+        cv2.rectangle(img, (x, y), (x + w, y+h), (0, 255, 0), 2)
+        text = "Moving Object detected"
+        print(text)
+        cnt = cnt+1
+        print(cnt)
+    cv2.putText(img, text, (10, 20), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 255), 2)
```
Comment (HarshCasper) [inline Python/Motion_Detection/script.py:38]: Draw the main logic of the code in a separate modular function

File: Python/Motion_Detection/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Installation
+
+> pip3 install -r requirements.txt
+
+
+## Usage
+- Run command python3 script.py
+
+## Output
+- After running the above command , your camera will be launched if motion is detected it will show label as "Moving Object detected" otherwise "Normal"
```
Comment (HarshCasper) [inline Python/Motion_Detection/README.md:13]: Add a GIF/Screenshot (hosted on Imgur) to demonstrate the usage

File: Python/Motion_Detection/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Motion detection script
+- This script detection any kind of motion or movement infront of the camera.
+
+## Installation
+
+> pip3 install -r requirements.txt
+
+
+## Usage
+- Run command python3 script.py
```
Comment (HarshCasper) [inline Python/Motion_Detection/README.md:10]: Add backticks around the command

File: Python/Motion_Detection/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Motion detection script
+- This script detection any kind of motion or movement infront of the camera.
```
Comment (HarshCasper) [inline Python/Motion_Detection/README.md:2]: Be slightly more descriptive

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Additionally, format the code using Black

**Comment by HarshCasper** [comment]
Closing the PR because of no updates.

**Comment by kskrao219** [comment]
> Closing the PR because of no updates.
Hello @HarshCasper , i know its been a long time since this PR was done , and you also suggested me the changes to do past month, but due to some academic events since one month i was unable to do so , i request you to kindly give me some more time so that i can do the changes you suggested and revert back to you.

**Pull request closed without merging**